spec/unit/writer/test_csv.rb

```diff
@@ -0,0 +1,80 @@
+require 'spec_helper'
+module Bmg
+  module Writer
+    describe Csv do
+
+      subject{ relation.to_csv(*args) }
+
+      let(:relation) {
+        Relation.new [
+          {id: "1", name: "Bernard Lambeau"},
+          {id: "2", name: "Yoann;Guyot"}
+        ]
+      }
+
+      context 'with default options and no output' do
+        let(:args){ [] }
+
+        it 'works' do
+          expected = <<-CSV.gsub(/^\s+/, "")
```

Comment:
Unless I'm missing something, you should be able to use the squiggly heredoc `<<~` here and drop the `gsub`.